src/Microsoft.ServiceFabric.Services/Communication/Client/ServicePartitionClient.cs

```diff
@@ -59,6 +59,11 @@ public class ServicePartitionClient<TCommunicationClient> : IServicePartitionCli
             this.targetReplicaSelector = targetReplicaSelector;
             this.lastRsp = null;
             this.retrySettings = retrySettings ?? new OperationRetrySettings();
+
+            ServiceTelemetry.CustomCommunicationClientUsageEvent(

```

Comment:
@suchiagicha  Could you please help whats the  best way to identify a custom communication client as this is called by Remoting client as well.

Reply:
Perhaps I should rename this to "CheckCustomCommunicationClientUsageEvent" as internally there is a check, and telemetry is only logged for non-remoting case.

Reply:
@vshan  User creates ServicePartitionClient for each call , which might be too much of data. Discussed with @amanbha  , we dont't need any client side telemetry. Service/Listener side telemetry should be enough.

Reply:
@amanbha  @suchiagicha  Ok sure, I have removed client side telemetry for ServicePartitionClient. Thanks